src/contract/operations.rs

```diff
@@ -278,8 +277,8 @@ pub trait Operation {
 pub struct Genesis {
     pub ffv: Ffv,
     pub schema_id: SchemaId,
-    pub chain: Chain,
-    pub alt_layer1: AltLayer1Set,
+    pub testnet: bool,
```

Comment:
this change seems should belong to another PR, especially because we're still deciding (here https://github.com/RGB-WG/rgb-core/pull/183) if we really want to no more differentiate different test networks